Python/ceval_macros.h

```diff
@@ -384,6 +384,24 @@ _PyFrame_SetStackPointer(frame, stack_pointer)
 /* Tier-switching macros. */
 
 #ifdef _Py_JIT
+#if Py_TAIL_CALL_INTERP
+#define GOTO_TIER_TWO(EXECUTOR)                        \
+do {                                                   \
+    OPT_STAT_INC(traces_executed);                     \
+    jit_func jitted = (EXECUTOR)->jit_code;            \
+    next_instr = jitted(frame, stack_pointer, tstate); \
+    Py_DECREF(tstate->previous_executor);              \
+    tstate->previous_executor = NULL;                  \
+    frame = tstate->current_frame;                     \
+    if (next_instr == NULL) {                          \
+        next_instr = frame->instr_ptr;                 \
+        stack_pointer = _PyFrame_GetStackPointer(frame); \
+        return _TAIL_CALL_error(TAIL_CALL_ARGS);       \
```

Comment:
This is the only line that's different, right? Could we just do `JUMP_TO_LABEL(error)` for both?

We want this to be a tail call anyways, I think.

Reply:
By the way the implementation of `GOTO_TIER_TWO` changed slightly on JIT builds. Check out what's on `main` currently and do that.

Reply:
Oh right I forgot. Thanks!